apis/projectcontour/v1alpha1/httploadbalancer.go

```diff
@@ -78,6 +78,8 @@ type TLS struct {
 
 // Route contains the set of routes for a virtual host
 type Route struct {
+	// Includes allow for specific routing configuration to be appended to another HTTPLoadBalancer in another namespace
+	Includes []Include `json:"includes,omitempty"`
```

Comment:
I don't think this should be here -- a route doesn't include another route, routes (plural) can be included from another httplb document at the spec level.